08-rails/routes-controllers.md

```diff
@@ -45,6 +45,77 @@ Let's break down this code:
   - **request method**: `get` in this example
   - **path**: what will be matched with the URL in the HTTP request, `"/books"` in this example
   - **controller and action**: defining the controller and action, split by the `#`. `"books#index"` in this case this would point to the `index` method in the `BooksController` class.
+  
+### Naming Routes 
+
+A `routes.rb` file which lists all the CRUD operations would look like this:
+
+```ruby
+# config/routes.rb
+Rails.application.routes.draw do
+
+  get "/books/new", to: "books#new", as: "new_book"
+  post "/books", to: "books#create"
```

Comment:
This route should have `, as: "books"` to match the `rails routes` listing below.